nuxeo-coldstorage-web/ftest/features/coldstorage.feature

```diff
@@ -61,10 +62,13 @@ Feature: Cold Storage
     And I move the files to cold storage
     When I browse to the document with path "/default-domain/ws/sample1"
     Then I can see the file is stored in cold storage
+    And I cannot see the Remove button
     When I browse to the document with path "/default-domain/ws/sample2"
     Then I can see the file is stored in cold storage
+    And I cannot see the Remove button
     When I browse to the document with path "/default-domain/ws/sample3"
     Then I can see the file is stored in cold storage
+    And I cannot see the Remove button
```

Comment:
Do we really need to repeat this step three times in the same scenario? Don't all of them have the same Coldstorage facet?